pykube/http.py

```diff
@@ -88,6 +89,23 @@ def send(self, request, **kwargs):
 
         if "token" in config.user and config.user["token"]:
             request.headers["Authorization"] = "Bearer {}".format(config.user["token"])
+
+        elif "exec" in config.user:
+            exec_conf = config.user["exec"]
+            if "command" in exec_conf:
+
+                for env_var in exec_conf.get('env', []):
```

Comment:
This can fail if `env` does exist in the `exec_conf` but is `None` (as just happened to me), so the suggestion below should fix this.

```suggestion
                for env_var in exec_conf.get('env') or []:
```